### DISCUSSION TITLE
Nur maximal eine Referenzrechnung in ZUGFeRD erlaubt

### DISCUSSION THREAD

**Discussion by Busch62** [LandrixSoftware/XRechnung-for-Delphi#33] [General]
In intf.XRechnung gibt es diesen Code:

  //Nur maximal eine Referenzrechnung in ZUGFeRD erlaubt
  if (_Version in [TXRechnungVersion.XRechnungVersion_230_UNCEFACT,
                   TXRechnungVersion.XRechnungVersion_30x_UNCEFACT]) then
  if _Invoice.PrecedingInvoiceReferences.Count > 1 then
  begin
    Result := false;
    exit;
  end;

Wo kommt denn diese Einschränkung her, dass da nur eine PrecedingInvoice erlaubt ist?
Wenn ich die obige Prüfung weglasse und in TXRechnungInvoiceAdapter301.SaveDocumentUNCEFACT entsprechend nicht nach der ersten PrecedingInvoiceReference abbreche, scheint trotzdem eine gültige Zufgerd-Rechnung erzeugt zu werden. Der Mustang-Validator meckert jedenfalls nicht.

**Comment by landrix** [comment]
also laut XRechnung 3.0.2 ist das im CII Format leider nicht der Fall. Da hier nur XRechnung erstellt wird, kann ich das noch nicht ändern. Vielleicht wird es dann in Version 4 der Fall sein
